pkg/runtime/cli.go

```diff
@@ -248,22 +248,26 @@ func FromFlags() (*DaprRuntime, error) {
 		globalConfig = global_config.LoadDefaultConfiguration()
 	}
 
-	var resiliencyConfigs []resiliency_config.Resiliency
+	var resiliencyConfigs []*resiliency_v1alpha.Resiliency
 
 	switch modes.DaprMode(*mode) {
 	case modes.KubernetesMode:
 		namespace = os.Getenv("NAMESPACE")
-		resiliencyConfigs = resiliency_config.LoadKubernetesResiliency(*appID, namespace, operatorClient, log)
+		resiliencyConfigs = resiliency_config.LoadKubernetesResiliency(*appID, namespace, operatorClient)
 	case modes.StandaloneMode:
-		resiliencyConfigs = resiliency_config.LoadStandaloneResiliency(*componentsPath, *appID, log)
+		resiliencyConfigs = resiliency_config.LoadStandaloneResiliency(*componentsPath, *appID)
+	}
+	if resiliencyConfigs == nil {
+		resiliencyConfigs = []*resiliency_v1alpha.Resiliency{}
 	}
```

Comment:
Given that you're still checking this here and replacing with an empty array, why'd you change the method to return a nil instead of the empty array?